tracing-log/src/log_tracer.rs

```diff
@@ -0,0 +1,232 @@
+use crate::{format_trace, AsTrace};
+use log;
+use tracing_core::dispatcher;
+
+/// A simple "logger" that converts all log records into `tracing` `Event`s.
+///
+/// Can be initialized with:
+///
+/// * [`init`] if you want to convert all logs and do the filtering in a subscriber
+/// * [`init_with_filter`] if you know in advance a log level you want to filter
+///
+/// [`init`]: ../fn.init.html
+/// [`init_with_filter`]: ../fn.init_with_filter.html
+#[derive(Debug)]
+pub struct LogTracer {
+    ignore_crates: Box<[String]>,
```

Comment:
Why dis over `Vec<String>`?

Reply:
Once we actually construct a `LogTracer`, the vector would never grow again, so I thought changing it to a `Box<[String]>` indicated that more clearly. Theoretically, it is also slightly smaller since it lacks a capacity field, but since this is essentially a singleton that isn't really important.

Reply:
Interesting, hadn't considered that. 👍